src/account/PluginManagerInternals.sol

```diff
@@ -838,6 +838,347 @@ abstract contract PluginManagerInternals is IPluginManager {
         emit PluginUninstalled(plugin, onUninstallSuccess);
     }
 
+    function _replacePlugin(
```

Comment:
idk why this is indicated as diff, but I think you started from the previous commits. Please check this